docs/reference/default-theme-config.md

```diff
@@ -17,7 +17,7 @@ export default {
 }
 ```
 
-**The options documented on this page only apply to the default theme.** Different themes expect different theme config. When using a custom theme, the theme config object will be passed to the theme so the theme can define conditional behavior based on it.
+**The options documented on this page only apply to the default theme.** Different themes expect different theme configuration options. When using a custom theme, the theme config object will be passed to the theme so the theme can define conditional behavior based on it.
```

Comment:
the earlier one was already clear IMO

Reply:
Well "config" is a bit more vague and also not an actual word in proper English - more like developer slang that might refer to some insider knowledge. I didn't really understand what kind of configuration it referred to when I first read it. I thought "configuration options" would be a more accurate term.

But I can skip this change if necessary.